scripts/validate-notebooks.ps1

```diff
@@ -23,7 +23,9 @@
 [CmdletBinding()]
 Param(
     [Parameter(Position=1)]
-    $Notebook = ""
+    $Notebook = "",
+    [double]$StartIndex = -1.0,
+    [double]$EndIndex = -1.0
```

Comment:
Could you please document the new parameters, similar to the `.PARAMETER Notebook` documentation on lines 17-19? This will help us remember what they do a year from now :-)